src/main/java/org/sqlite/jdbc3/JDBC3DatabaseMetaData.java

```diff
@@ -2243,4 +2236,24 @@ public String toString() {
     protected void finalize() throws Throwable {
         close();
     }
+    
+    /**
+     * Follow rules in <a href="https://sqlite.org/lang_keywords.html">SQLite Keywords</a>
+     * @param name Identifier name
+     * @return Unquoted identifier
+     */
+    private String unquoteIdentifier(String name) {	
+    	if (name == null) return name;
+    	name = name.trim();
+        if (name.length() > 2 && (
```

Comment:
You can use a regular expression like `^(['"\[])(.*)\1$`, then extract group 2, but the current code might be easier to read.

Reply:
So it's ok to leave this as is